src/manager.spec.js

```diff
@@ -1413,18 +1413,19 @@ describe('AdManager', function() {
   });
 
   describe('#refreshSlot', function() {
-    var adSlot, stubSlot;
+    var adSlot, stubSlot, baseContainer;
 
     beforeEach(function() {
       var setupRefs = adSlotSetup();
       adSlot = setupRefs.adSlot1;
       stubSlot = setupRefs.stubSlot;
+      baseContainer = setupRefs.baseContainer;
       TestHelper.stub(adManager, 'refreshSlots');
       adManager.options.amazonEnabled = false;
     });
 
     afterEach(function() {
-      document.body.innerHTML = "";
+      $(baseContainer).remove();
```

Comment:
`$` is non-standard and could potentially cause issues in the future or yell if this is ever linted